tap_xenon.tex

```diff
@@ -0,0 +1,418 @@
+Generic \gls{PWR} during steady-state operation at a constant neutron flux 
+level reaches $^{135}$Xe/$^{135}$I equilibrium for that reactor power in about 
+40 to 60 hours. When the power level is decreased, the neutron flux is reduced, 
+less $^{135}$Xe is transmuted ('burned') to $^{136}$Xe, and the balance shifts 
+to the larger $^{135}$Xe concentration. For the worst-case scenario for the 
+\gls{PWR} (power was reduced to 0\%) the xenon concentration peaks in about 
+8-9 hours after shutdown (depends on the neutron spectrum). Based on this 
+knowledge of \gls{PWR} poisoning, we selected following worst-case load curve 
+(see figure~\ref{fig:pwr_keff}, 
+\ref{fig:tap_keff}; red line):
+\begin{enumerate}
+	\item Startup with fresh fuel and operating on 
+	100\% power level for 96 hours to reach $^{135}$Xe/$^{135}$I balance;
+	\item Instantaneous variation of the power from 100\% to 0\%;
+	\item Shutdown for 7.66 hours to reach the $^{135}$Xe peak;
+	\item Instantaneous power boost from 0\% to 100\%, then operation on 
+	this level for 16 hours.
+\end{enumerate}
+This scenario can be considered as backing up solar power with nuclear on a 
+high-solar penetration grid (e.g., in California).
+
+\section{Results and Analysis}
+Using the methodology described previously, the \gls{PWR} assembly and 
+the \gls{TAP} full core depletion analysis was performed. 
+We used 15-min power load curve resolution during the active phase of the 
+transient to capture rapid changes in multiplication factor and isotopic 
+composition. All results herein were obtained using built-in Serpent 2 
+capabilities for fuel depletion calculations.
+
+\subsection{Multiplication Factor and Xenon Equilibrium}
+Fig.~\ref{fig:pwr_keff} shows the infinite multiplication factor 
+($k_{\infty}$) evolution for the \gls{PWR} assembly during postulated 
+transient. As expected, $k_{\infty}$ dropped  after shutdown from 1.120 to 
+1.105 ($-1500$ pcm) because $^{135}$I decayed to $^{135}$Xe, and xenon 
+poisoned the core. The multiplication factor reached local minimum  
+$\approx7$ hours after shutdown. Notably, after power level growth from 
+0\% back to 100\%, 
+$k_{\infty}$ reached even larger value than it was before shutdown ($\approx 
+1.1234$, $+300$ pcm). The imbalance between $^{135}$I production and 
+$^{135}$Xe 'burning' is a main reason of this reactivity boost.
+\begin{figure}[htbp!] % replace 't' with 'b' to force it to be on the bottom
+	\centering
+	\includegraphics[width=1.03\linewidth]{pwr_keff_zoomed.png}
+		\vspace{-0.25in}
+	\caption{Multiplication factor for worst-case load curve for the \gls{PWR} 
+	immediately after shutdown (confidence interval $\sigma\pm20$pcm shaded).}
+		\vspace{-0.05in}
+	\label{fig:pwr_keff}
+\end{figure}
+
+Fig.~\ref{fig:tap_keff} demonstrates the effective multiplication factor 
+($k_{eff}$) dynamics in the same transient for full-core \gls{TAP} \gls{MSR} 
+model without any online \gls{FP} removal. Surprisingly, $k_{eff}$ slowly 
```

Comment:
If the only thing that changed was power (we did not touch temperature), and the Xe-135 barely changed, what isotope is causing this increase in keff?

Reply:
1. ANS Meeting Summary template uses Fig.
2. I don't know, some additional simulations needed to figure it out. Most likely, there are some other poisons (not Xe). But overall reactivity swing is much smaller for TAP, that's why I ignoring this increase of reactivity, this jump is really small.